res/skins/Deere/effect_parameter_knob.xml

```diff
@@ -19,16 +19,19 @@
         <Layout>horizontal</Layout>
         <Size>me,20f</Size>
         <Children>
-          <KnobComposed>
-            <TooltipId><Variable name="TooltipId"/></TooltipId>
+          <EffectParameterKnobComposed>
             <Size>20f,20f</Size>
             <Knob>knob_small.svg</Knob>
             <MinAngle>-230</MinAngle>
             <MaxAngle>50</MaxAngle>
+            <EffectRack><Variable name="EffectRack"/></EffectRack>
+            <EffectUnit><Variable name="EffectUnit"/></EffectUnit>
+            <Effect><Variable name="Effect"/></Effect>
+            <EffectParameter><Variable name="EffectParameter"/></EffectParameter>
```

Comment:
Getting that information from the connected CO would require writing new string parsing functions. I'd rather keep reusing the ones that already exist.